src/main/java/it/pagopa/pm/gateway/controller/CreditCardPaymentController.java

```diff
@@ -80,15 +81,16 @@ private ResponseEntity<StepZeroResponse> buildResponseStep0(StepZeroResponse ste
     }
 
     @PostMapping(REQUEST_PAYMENTS_RESUME_METHOD)
-    public ResponseEntity<String> resumeCreditCardPayment(@RequestHeader(required = false, value = MDC_FIELDS) String mdcFields,
-                                                          @PathVariable String requestId,
-                                                          @RequestBody CreditCardResumeRequest request) {
+    public ResponseEntity<VposResumeMethodResponse> resumeCreditCardPayment(@RequestHeader(required = false, value = MDC_FIELDS) String mdcFields,
+                                                                            @PathVariable String requestId,
```

Comment:
Given that a UUID is expected, can you change it to
```suggestion
                                                                            @PathVariable UUID requestId,
```
?

Reply:
Below I used a String.valueOf because otherwise I had to change all the methods in the service.
https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/200/commits/571d57cf26e2e5d171fd71dc93741e3257a43d39